docs/project/governance.md

```diff
@@ -0,0 +1,96 @@
+# pyodide governance and decision-making
+
+The purpose of this document is to formalize the governance process used by the
+pyodide project, to clarify how decisions are made and how the various
+elements of our community interact.
+This document establishes a decision-making structure that takes into account
+feedback from all members of the community and strives to find consensus, while
+avoiding any deadlocks.
+
+Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+## Roles And Responsibilities
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the {ref}`how_to_contibute`.
+
+### Community members team
+
+The community members team is composed of community members who have permission on
+Github to label and close issues. Their work is
```

Comment:
Not sure about the name for this team. iodode has "community members team", the other name could be "triage team".

Another side effect from being in this team is that it would be possible to request review on PRs from its members via Github UI. That feature doesn't work for users not part of the org.

Reply:
I think "community members team" sounds friendlier, "triage team" sounds gory and like it requires medical training.